WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java

```diff
@@ -264,6 +278,10 @@ public void onDataLoaded(boolean isEmpty) {
                             updateComments(isEmpty, false);
                         } else if (mRestorePosition > 0) {
                             mRecyclerView.scrollToPosition(mRestorePosition);
+                        } else if (mCommentId > 0) {
+                            // Scroll to the commentId once if it was passed to this activity
+                            scrollToCommentId(mCommentId);
```

Comment:
It worked well enough IMO. 



> The if (isEmpty || !mHasUpdatedComments) check above means the comment won't be scrolled to until after the comments have been updated, which may be several seconds after the activity appears.

The delay never bothered me, does it bug you enough to not have this integration?



> Only the first 20 comments will be retrieved when the activity appears, and there's no guarantee the desired comment will be in these first 20 (it may not be retrieved until the user performs an infinite scroll).

I doubt many posts have more than 20 comments, but good point.